custom_components/tahoma/const.py

```diff
@@ -8,8 +8,18 @@
 from homeassistant.components.sensor import DOMAIN as SENSOR
 from homeassistant.components.switch import DOMAIN as SWITCH
 
+CONF_HUB = "hub"
 DOMAIN = "tahoma"
 
+DEFAULT_HUB = "Somfy TaHoma"
+SUPPORTED_ENDPOINTS = {
+    "Cozytouch": "https://ha110-1.overkiz.com/enduser-mobile-web/enduserAPI/",
+    "Somfy TaHoma": "https://tahomalink.com/enduser-mobile-web/enduserAPI/",
+    "Somfy Connexoon IO": "https://tahomalink.com/enduser-mobile-web/enduserAPI/",
+    "Somfy Connexoon RTS": "https://ha201-1.overkiz.com/enduser-mobile-web/enduserAPI/",
+    "Rexel": "https://ha112-1.overkiz.com/enduser-mobile-web/enduserAPI/",
```

Comment:
Would be nice later to add this within pyhoma.

Reply:
@tetienne good one! Would be a good improvement in the future.